src/argilla/client/feedback/dataset.py

```diff
@@ -234,7 +140,7 @@ def __init__(
         self,
         *,
         fields: List[FieldSchema],
-        questions: List[AllowedQuestionTypes],
+        questions: List[Union[TextQuestion, RatingQuestion]],
```

Comment:
Maybe we should use here `argilla.client.feedback.schemas.AllowedQuestionTypes`?

Reply:
Yes, it's like that in the PR I created on top of this to add also the new question types 👍🏻